velox/docs/functions/spark/map.rst

```diff
@@ -47,3 +47,25 @@ Map Functions
     Returns the size of the input map. Returns null for null input
     if :doc:`spark.legacy_size_of_null <../../configs>` is set to false.
     Otherwise, returns -1 for null input.
+
+.. spark:function:: transform_keys(map(K1,V), function(K1,V,K2)) -> map(K2,V)
+
+    Returns a map that applies ``function`` to each entry of ``map`` and transforms the keys.::
```

Comment:
Shall we mention the behavior is consistent with EXCEPTION for MAP_KEY_DEDUP_POLICY in Spark, and is any change needed with https://github.com/facebookincubator/velox/pull/9562?

Reply:
don't think need, just like other `map*` functions description we already have in spark velox sheet.

we just need wait the fix #9562 ready and then I can do follow up to fix `transformKeys` support `EXCEPTION` option with this issue https://github.com/facebookincubator/velox/issues/10219

Reply:
nit: It would be better to document the behavior when there are duplicate keys.
nit: add a white-space between `.` and `::`.

Reply:
@gaoyangxiaozhu Would you follow up this nit? Thanks.

Reply:
@gaoyangxiaozhu, could you check the above minor comments?

Reply:
> we just need wait the fix https://github.com/facebookincubator/velox/pull/9562 ready and then I can do follow up to fix transformKeys support EXCEPTION option with this issue https://github.com/facebookincubator/velox/issues/10219

Does it still make sense to register the Presto implementation to Spark if the implementations will become different in the future?